dist/index.html

```diff
@@ -17,101 +16,96 @@
         <p class="tag-text">Antipasti</p>
       </section>
-      <section class="tag-card" id="starter">
+      <section class="tag-card" id="starter" tabindex="0">
         <p class="tag-text">Starter</p>
       </section>
-      <section class="tag-card" id="appetizer">
+      <section class="tag-card" id="appetizer" tabindex="0">
         <p class="tag-text">Appetizer</p>
       </section>
-      <section class="tag-card" id="antipasto">
+      <section class="tag-card" id="antipasto" tabindex="0">
         <p class="tag-text">Antipasto</p>
       </section>
-      <section class="tag-card" id="hor d'oeuvre">
+      <section class="tag-card" id="hor d'oeuvre" tabindex="0">
         <p class="tag-text">Hor d'oeuvre</p>
       </section>
-      <section class="tag-card" id="main course">
+      <section class="tag-card" id="main course" tabindex="0">
         <p class="tag-text">Main course</p>
       </section>
-      <section class="tag-card" id="sauce">
+      <section class="tag-card" id="sauce" tabindex="0">
         <p class="tag-text">Sauce</p>
       </section>
-      <section class="tag-card" id="side dish">
+      <section class="tag-card" id="side dish" tabindex="0">
         <p class="tag-text">Side dish</p>
       </section>
-      <section class="tag-card" id="brunch">
+      <section class="tag-card" id="brunch" tabindex="0">
         <p class="tag-text">Brunch</p>
       </section>
-      <section class="tag-card" id="morning meal">
+      <section class="tag-card" id="morning meal" tabindex="0">
         <p class="tag-text">Morning Meal</p>
       </section>
-      <section class="tag-card" id="salad">
+      <section class="tag-card" id="salad" tabindex="0">
         <p class="tag-text">Salad</p>
       </section>
-      <section class="tag-card" id="condiment">
+      <section class="tag-card" id="condiment" tabindex="0">
         <p class="tag-text">Condiment</p>
       </section>
-      <section class="tag-card" id="dip">
+      <section class="tag-card" id="dip" tabindex="0">
         <p class="tag-text">Dip</p>
       </section>
-      <section class="tag-card" id="spread">
+      <section class="tag-card" id="spread" tabindex="0">
         <p class="tag-text">Spread</p>
       </section>
     </section>
-    <section class="recipe-area">
+    <section class="recipe-area" tabindex="0">
       <!-- cards go here  -->
     </section>
-    <section id="recipeCardBlowup" class="recipe-card-blowup hidden">
-      <section class="icon-section">
-        <img class="icon bookmark icon-bookmark hidden" src="./images/bookmark-regular.svg" alt="bookmark icon" />
-        <img class="icon bookmark solid-bookmark hidden" src="./images/bookmark-solid.svg" alt="solid bookmark icon" />
-        <div class="spacer"></div>
-        <img class="icon close" src="./images/x-solid.svg" alt="close icon" />
-      </section>
+    <section id="recipeCardBlowup" class="recipe-card-blowup hidden" tabindex="0" >
```

Comment:
@MaryBruff I noted this is missing the `<section class="icon-section"> tag`, this is likely to mess up the styling of the bookmark and close tags if not incorporated.